cache/README.md

```diff
@@ -0,0 +1,192 @@
+# Cache
+
+**Cache** is an in-memory caching library for Kotlin, primarily targeted at mobile.
```

Comment:
Let's try to open with a gist of what important for someone who stumbled across this library:

Cache is a pure Kotlin in-memory caching library.

Primary features:
Time base eviction
Size based eviction
Cache loaders

Reply:
Thanks. How about:

**Cache** is an in-memory caching library written in 100% Kotlin. Primay features include:

* Time-based evictions (expirations)
* Size-based evictions
* Cache loader